distro/exporter.go

```diff
@@ -74,18 +74,40 @@ func newOTLPTracesExporter(_ logr.Logger, c *exporterConfig) (trace.SpanExporter
 		)
 	}
 
+	headers := make(map[string]string)
+	if c.AccessToken != "" {
+		headers["X-Sf-Token"] = c.AccessToken
+	}
+	shouldUseInsecure := noneEnvVarSet(otelExporterOTLPEndpointKey, otelExporterOTLPTracesEndpointKey, splunkRealmKey)
```

Comment:
```suggestion
	isLocalCollector := noneEnvVarSet(otelExporterOTLPEndpointKey, otelExporterOTLPTracesEndpointKey, splunkRealmKey)
```

I think would be a lot more readable.